tests/components/cloud/test_http_api.py

```diff
@@ -398,8 +398,34 @@ async def test_websocket_status(
             "google_enabled": True,
             "google_entity_configs": {},
             "google_secure_devices_pin": None,
-            "google_default_expose": None,
-            "alexa_default_expose": None,
+            "google_default_expose": [
```

Comment:
This is not a good change. We need to be able to update the default exposed domains if we add new domains and that's now not possible.